core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -1,6 +1,6 @@
 <script type="text/ng-template" id="summaryTile/exploration">
   <md-card class="oppia-activity-summary-tile" ng-class="{'small-width': !isWindowLarge}">
-    <a ng-href="<[getExplorationLink()]>" target="<[openInNewWindow ? '_blank' : '_self']>">
+      <a ng-if="getExplorationLink()" ng-href="<[getExplorationLink()]>" target="<[openInNewWindow ? '_blank' : '_self']>"></a>
```

Comment:
Deindent by 2, like it was before (so that it is relatively indented by 2 from the previous line). Deindent the following lines by 2 as well.